### PR TITLE
feat: adds Jupyter notebook to JSS

### PR DESCRIPTION
**Pull request by telpirion**

### CODE REVIEW

File: notebook/gen_ai_jss.ipynb [hunk 1]
```diff
@@ -0,0 +1,818 @@
...
+   },
+   "source": [
+    "### Objective\n",
+    "\n",
+    "In this tutorial, you learn how to create a Cloud Function process that transcribes characters from a PDF, stores the complete PDF text in a Storage bucket, summarizes the PDF, and then upserts the document data (summary, complete text, URI) into a BigQuery table.\n",
+    "\n",
+    "This tutorial uses the following Google Cloud services and resources:\n",
+    "\n",
+    "- Vertex AI Generative AI\n",
+    "- CLoud BigQuery\n",
```
Comment (yil532) [inline notebook/gen_ai_jss.ipynb:98]: Typo: Cloud BigQuery

File: notebook/gen_ai_jss.ipynb [hunk 2]
```diff
@@ -0,0 +1,818 @@
...
+    "id": "BF1j6f9HApxa"
+   },
+   "source": [
+    "## Before you begin\n",
+    "\n",
+    "### Set up your Google Cloud project\n",
+    "\n",
+    "This notebook assumes that you have already deployed this solution using either the [Terraform script]() **TODO: fix target for link** or using the [Solutions console](https://console.cloud.google.com/products/solutions/catalog). During this deployment, several actions required to run this solution were performed on your behalf:\n",
+    "\n",
+    "1. The [Cloud Function](https://console.cloud.google.com/functions/list) was deployed.\n",
```
Comment (yil532) [inline notebook/gen_ai_jss.ipynb:229]: I would prefer to remove the 1., 2., 3., 4. to make it more aesthetically appealing.

File: notebook/gen_ai_jss.ipynb [hunk 3]
```diff
@@ -0,0 +1,818 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "sBCra4QMA2wR"
+   },
+   "source": [
+    "### Authenticate your Google Cloud account\n",
+    "\n",
+    "Depending on your Jupyter environment, you may have to manually authenticate. Follow the relevant instructions below."
```
Comment (yil532) [inline notebook/gen_ai_jss.ipynb:301]: Maybe users wonder where they can run this notebook. Cloud shell could be an easy option. How about list out the few choices to run this colab?

File: notebook/gen_ai_jss.ipynb [hunk 4]
```diff
@@ -0,0 +1,818 @@
...
+      "/ [1 files][196.1 KiB/196.1 KiB]                                                \n",
+      "Operation completed over 1 objects/196.1 KiB.                                    \n"
+     ]
+    }
+   ],
+   "source": [
+    "filename = '9404002v1'\n",
+    "file_uri = f'gs://arxiv-dataset/arxiv/cmp-lg/pdf/9404/{filename}.pdf'\n",
+    "\n",
+    "# Create a local folder and download some test PDFs\n",
```
Comment (yil532) [inline notebook/gen_ai_jss.ipynb:437]: Can we sneak peak the files here? Can insert an image here.

Comment (yil532) [reply notebook/gen_ai_jss.ipynb:437]: We want a before & after experience. (PDF v.s. output text)

File: notebook/gen_ai_jss.ipynb (hunk 2 above)
Comment (telpirion) [reply notebook/gen_ai_jss.ipynb:229]: Done.

File: notebook/gen_ai_jss.ipynb (hunk 1 above)
Comment (telpirion) [reply notebook/gen_ai_jss.ipynb:98]: Done.

File: notebook/gen_ai_jss.ipynb (hunk 3 above)
Comment (telpirion) [reply notebook/gen_ai_jss.ipynb:301]: Good point!

They should ideally run this in Workbench. However, it is *possible* that they would try to run it directly from GitHub. We should probably think through where / how users access this notebook.

File: notebook/gen_ai_jss.ipynb (hunk 4 above)
Comment (telpirion) [reply notebook/gen_ai_jss.ipynb:437]: We can probably show part of one image as a "before." These PDF files are usually > 7 pages long.

File: notebook/gen_ai_jss.ipynb (hunk 3 above)
Comment (telpirion) [reply notebook/gen_ai_jss.ipynb:303]: Revised comment.

File: notebook/gen_ai_jss.ipynb (hunk 4 above)
Comment (telpirion) [reply notebook/gen_ai_jss.ipynb:424]: Let's add an image as a v2 enhancement.

File: notebook/gen_ai_jss.ipynb [hunk 5]
```diff
@@ -0,0 +1,670 @@
+{
```
Comment (nicain) [inline notebook/gen_ai_jss.ipynb:1]: <p>I didn't realize that we had planned on deployment through a script. If that is still planned, we can still use deploy.sh, correct?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/terraform-genai-doc-summarization/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (nicain) [inline notebook/gen_ai_jss.ipynb:1]: <p>Nice!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/terraform-genai-doc-summarization/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (nicain) [inline notebook/gen_ai_jss.ipynb:1]: <p>Good thing I still pass this through to the second call! https://github.com/GoogleCloudPlatform/terraform-genai-doc-summarization/blob/c86b3d6505ac06fe1b0bfb99d91b7593f3caa39f/webhook/main.py#L115</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/terraform-genai-doc-summarization/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

**Review by nicain — APPROVED** [review]

**Pull request merged**